documentation/release_notes.rst

```diff
@@ -8,6 +8,21 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.1.1
+===============
+New Features
+------------
+- Improved ``sort`` algorithm performance for the arithmetic data types with ``std::less`` or ``std::greater`` comparison operator and DPC++ policy.
+- AOT compilation for CPU devices is now possible with the use of ``ONEDPL_USE_AOT_COMPILATION`` cmake option.
+
+Fixes Issues
+------------
+- Fixed ``Device name missing`` error occurring with the following cmake options: ``ONEDPL_USE_AOT_COMPILATION=ON``, ``ONEDPL_DEVICE_BACKEND=GPU`` or not specified and ``CMAKE_BUILD_TYPE`` is not specified.
+- Fixed an error that caused segmentation faults in ``transform_reduce``, ``minmax_element``, and related algorithms when ran on CPU devices. 
+- Fixed a compilation error in ``transform_reduce``, ``minmax_element``, and related algorithms on FPGAs.
+- Fixed an issue with oneDPL permutation iterator. 
+- Fixed an issue with 64-bit integer types with DPC++ policy.
```

Comment:
How have these issues impacted (maybe potentially) user code? That should be clear enough so that users could correlate to their problems. If no impact on user code, why is it worth mentioning?

Reply:
For (1), I suggest "Fixed permutation_iterator to support C-style array as a permutation map"

Reply:
Added the suggested.